package.nls.json

```diff
@@ -31,5 +31,8 @@
     "configuration.java.test.config.vmArgs.description": "Specify the extra options and system properties for the JVM",
     "configuration.java.test.config.args.description": "Specify the command line arguments which will be passed to the test runner",
     "configuration.java.test.config.env.description": "Specify the extra environment variables when running the tests",
-    "configuration.java.test.config.sourcePaths.description": "Specify extra source paths when debugging the tests"
+    "configuration.java.test.config.sourcePaths.description": "Specify extra source paths when debugging the tests",
+    "contributes.viewsWelcome.inLightWeightMode": "No test cases are listed because the Java Language Server is currently running in [LightWeight Mode](https://aka.ms/vscode-java-lightweight). To show test cases, click on the button to switch to Standard Mode.\n[Switch to Standard Mode](command:java.server.mode.switch?%5B%22Standard%22,true%5D)",
+    "contributes.viewsWelcome.noProjectWithProjectManagerInstalled": "No folder opened in Visual Studio Code. You can [open a Java project](command:_java.project.open), or create a new Java project by clicking the button below.\n[Create Java Project](command:java.project.create)",
+    "contributes.viewsWelcome.noProjectWithOutProjectManagerInstalled": "No folder opened in Visual Studio Code."
```

Comment:
Can we provide a shortcut to open a folder? Maybe it's not that feasible as commands are different per os.

Reply:
We cannot since things in package.json are static.

Reply:
so I'm really interested in how vscode implements that.

Reply:
https://github.com/microsoft/vscode/blob/413963c489fafa5163b5d6513731c7953de07fb3/src/vs/workbench/contrib/files/browser/explorerViewlet.ts#L297-L312

Reply:
I just created a issue to request a unified command id:
https://github.com/microsoft/vscode/issues/117748